### DISCUSSION TITLE
Record Audio from Browser

### DISCUSSION THREAD

**Discussion by jankapunkt** [openqda/openqda#131] [Ideas]
A colleague of us came up with the idea to directly record from browser into an audio file and transcribed text, which are to be uploaded.

Easy to implement:

- https://developer.mozilla.org/en-US/docs/Web/API/SpeechRecognition
- https://developer.mozilla.org/en-US/docs/Web/API/MediaRecorder

**Comment by kadewe** [comment]
but what are the real use cases? I think you should ALWAYS make a recording with a dedicated audio recorder? I just don't trust a webpage based audio recorder - what happens if the browser crashes?
